pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -3702,6 +3702,11 @@ public double getLoadBalancerBandwidthOutResourceWeight() {
             doc = "SSL Factory plugin configuration parameters used by internal client.")
     private String brokerClientSslFactoryPluginParams = "";
 
+    @FieldContext(category = CATEGORY_SERVER, dynamic = true,
+            doc = "Topic name contains cluster, not allow be created automatically."
+    )
+    private boolean enableCreateLegacyTopic = false;
```

Comment:
The config name `enableCreateLegacyTopic` is not optimal. Since this is related to `allowAutoTopicCreation`, it's better to share that prefix, for example `allowAutoTopicCreationWithLegacyNamingScheme` would be a more accurate config name. It should also be specified in the documentation that this config is only used when `allowAutoTopicCreation=true`.

Reply:
ok, I will modify it

Reply:
Looking forward to that.

Reply:
I have revised and perfected it